### PR TITLE
✨ Add footer with logos/CR/CGU

### PR DESCRIPTION
**Pull request by Koraltaron**

### CODE REVIEW

File: client/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+function Footer() {
+  return (
+    <>
+      <footer>
+        <a href="/">&copy;CopyRight</a>
+        <section className="logo">
+          <a id="logo-insta" href="/">
+            {" "}
+            <img
```
Comment (Jeremy-Mrz) [inline client/src/components/Footer.tsx:11]: Why {" "} ?

File: client/src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              src="https://cdn-icons-png.flaticon.com/512/87/87390.png"
+              alt="logo-instagram"
+            />{" "}
+          </a>
+          <a id="logo-fb" href="/">
+            {" "}
+            <img
```
Comment (Jeremy-Mrz) [inline client/src/components/Footer.tsx:18]: Again

File: client/src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+              src="https://cdn-icons-png.flaticon.com/512/59/59439.png"
+              alt="logo-facebook"
+            />{" "}
+          </a>
```
Comment (Jeremy-Mrz) [inline client/src/components/Footer.tsx:22]: Again

File: client/src/components/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+          <a id="logo-gh" href="/">
+            {" "}
+            <img
```
Comment (Jeremy-Mrz) [inline client/src/components/Footer.tsx:25]: Again

File: client/src/components/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+              src="https://cdn-icons-png.flaticon.com/512/25/25231.png"
+              alt="logo-github"
+            />{" "}
+          </a>
```
Comment (Jeremy-Mrz) [inline client/src/components/Footer.tsx:29]: Again

**Review by Jeremy-Mrz — CHANGES_REQUESTED** [review]

File: client/src/components/Footer.tsx (hunk 1 above)
Comment (FAlex357) [reply client/src/components/Footer.tsx]: Change are done !

**Review by Jeremy-Mrz — CHANGES_REQUESTED** [review]
Changes are good. But you have a merge conflict to deal with !

**Review by Jeremy-Mrz — APPROVED** [review]
Well done !

**Comment by FAlex357** [comment]
Conflict are resolved

**Pull request merged**